### PR TITLE
First section

### PR DESCRIPTION
**Pull request by katsu223**

### CODE REVIEW

File: resources/css/index.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Yeseva One";
+  src: url(../fonts/YesevaOne.otf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+body, html {
```
Comment (magicsd) [inline resources/css/index.css]: Line break is missing, fix here and below

File: resources/css/index.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  margin: 0;
+  padding: 0;
+}
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.container h1 {
+  font-family: "Yeseva One", sans-serif;
```
Comment (magicsd) [inline resources/css/index.css]: Sort properties alphabetically

File: resources/css/index.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  display: block;
+  text-align: center;
+  font-size: 90px;
+  color: white;
```
Comment (magicsd) [inline resources/css/index.css]: Use hex colors: `#fff`

File: resources/css/index.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  margin: 0;
+  font-weight: 400;
+  text-shadow:  1px 1px 2px black;
+}
+.background-overlay .container h1:nth-child(2) {
+  margin-bottom: 30px;
+}
+
+div.back-1 {
+    background-image: url(../images/background/Bg.png);
```
Comment (magicsd) [inline resources/css/index.css]: Something wrong with indentation

File: resources/css/index.css [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  color: white;
+  margin: 0;
+  font-weight: 400;
+  text-shadow:  1px 1px 2px black;
+}
+.background-overlay .container h1:nth-child(2) {
+  margin-bottom: 30px;
+}
+
+div.back-1 {
```
Comment (magicsd) [inline resources/css/index.css]: It's not good practice to use tags for styling, use classes

File: resources/css/index.css [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+.layout-positioner::after {
+  content: "";
+  display: table;
+  clear: both;
+ }
+
+ div.navbar {
+  background: url(../images/background/shadow.png) no-repeat center center;
+  width: 1600px;
+  height: 159px;
```
Comment (magicsd) [inline resources/css/index.css]: Height usually should be allocated automatically by content and paddings. Here and everywhere.

File: resources/css/index.css [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  margin: 0;
+  border: 1px solid white;
+  background-color: rgba(255, 255, 255, 0.2);
+ }
+
+.container .links {
+  display: inline-block;
+}
+
+.container {
```
Comment (magicsd) [inline resources/css/index.css]: Blank style, remove please

**Comment by katsu223** [comment]
http://katsu223.surge.sh/

File: index.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+                  <a href="#book">book a table</a>
+                  <a href="#menu">see the menu</a>
+                </div>
+            </div>
+          </section>
+      </article>
+    </main>
+  </div>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (magicsd) [reply index.html]: Line break should be after. Every file should end with break

File: resources/css/index.css (hunk 2 above)
Comment (magicsd) [reply resources/css/index.css]: `p` - also tag, call it may be `.header`, as context goes

File: resources/css/index.css [hunk 9]
```diff
@@ -16,31 +16,30 @@ body, html {
...
   margin-bottom: 30px;
 }
 
-div.back-1 {
-    background-image: url(../images/background/Bg.png);
-    width: 1600px;
-    height: 750px;
-    position: absolute;
-    top: 0px;
+.back-1 {
```
Comment (magicsd) [inline resources/css/index.css]: Unclear class name

File: index.html [hunk 10]
```diff
@@ -1,43 +1,30 @@
...
-                <div class="links">
-                  <a href="#book">book a table</a>
-                  <a href="#menu">see the menu</a>
-                </div>
-            </div>
-          </section>
+        <section class="background-overlay">
+          <div class="back-1"></div>
+          <div class="container">
+            <p>the right ingredients</p>
```
Comment (magicsd) [inline index.html]:
```html
<div class="main-title">
  the right ingredients <br>
  for the right food
</div>
```

File: index.html [hunk 11]
```diff
@@ -1,43 +1,30 @@
...
+            <p>for the right food</p>
+              <img src="resources/images/background/divider.png">
```
Comment (magicsd) [inline index.html]:
```html
<img class="divider" src="resources/images/background/divider.png">
```

File: index.html (hunk 10 above)
Comment (katsu223) [reply index.html]: fixed

File: index.html (hunk 11 above)
Comment (katsu223) [reply index.html]: fixed

File: index.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (katsu223) [reply index.html]: fixed

File: resources/css/index.css (hunk 2 above)
Comment (katsu223) [reply resources/css/index.css]: fixed

File: resources/css/index.css (hunk 9 above)
Comment (katsu223) [reply resources/css/index.css]: fixed

File: index.html [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+              <li><a href="#reservations">Reservations</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main>
+      <article class="content">
+        <section class="background-overlay">
+          <div class="background-1"></div>
```
Comment (magicsd) [inline index.html]: Unclear class name

File: resources/css/index.css [hunk 13]
```diff
@@ -0,0 +1,211 @@
+
```
Comment (magicsd) [inline resources/css/index.css]: Unnecessary blank line

File: resources/css/index.css [hunk 14]
```diff
@@ -0,0 +1,211 @@
...
+  margin-bottom: 30px;
+}
+
+.background-1 {
+  background-image: url(../images/background/Bg.png);
+  background-position: 50% 0px;
+  width: 1600px;
+  height: 750px;
+  position: absolute;
+  background-attachment: fixed;
```
Comment (magicsd) [inline resources/css/index.css]: Again: sort properties alphabetically in all styles.

File: resources/css/index.css [hunk 15]
```diff
@@ -0,0 +1,211 @@
...
+  font-weight: 700;
+  color: #fff;
+}
+
+.container .divider {
+  display: block;
+  margin: 0 auto;
+}
+
+.container .links a {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 16]
```diff
@@ -0,0 +1,211 @@
...
+  font-weight: 400;
+  text-shadow:  1px 1px 2px #000;
+}
+
+.container {
+  position: relative;
+  z-index: 2;
+}
+
+.layout-positioner h1 a {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 17]
```diff
@@ -0,0 +1,211 @@
...
+  float: right;
+ }
+
+ .navbar .main-menu {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+ }
+
+ .navbar .main-menu li {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 18]
```diff
@@ -0,0 +1,211 @@
...
+  display: inline-block;
+  margin:0px;
+  box-sizing: border-box;
+  margin-left: -4px;
+ }
+
+ .navbar .main-menu li:first-child {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 19]
```diff
@@ -0,0 +1,211 @@
...
+  margin-left: 0;
+ }
+
+ .navbar .main-menu li a {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 20]
```diff
@@ -0,0 +1,211 @@
...
+  display: block;
+  text-decoration: none;
+  color: #fff;
+  font-family: "GenB", sans-serif;
+  font-size: 21px;
+  padding: 44px 30px;
+  overflow: hidden;
+ }
+
+ .navbar .main-menu li:hover a {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 21]
```diff
@@ -0,0 +1,211 @@
...
+  float: right;
+ }
+
+ .social-media .social-menu {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+ }
+
+ .social-media .social-menu li {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 22]
```diff
@@ -0,0 +1,211 @@
...
+  display: inline-block;
+  margin:0px;
+  box-sizing: border-box;
+  margin-left: -4px;
+  padding: 36px 0;
+ }
+
+ .social-media .social-menu li a {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 23]
```diff
@@ -0,0 +1,211 @@
...
+  font-size: 21px;
+  color: #dab33e;
+  display: block;
+  text-decoration: none;
+  padding: 8px 5px;
+ }
+
+ .social-media social-menu li a:hover {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 24]
```diff
@@ -0,0 +1,211 @@
...
+  color: #f0c955;
+  text-shadow: 1px 1px 2px #000;
+ }
+
+ .social-media .social-menu li:first-child {
```
Comment (magicsd) [inline resources/css/index.css]: Again: prefer class names instead of tags for styling

File: resources/css/index.css [hunk 25]
```diff
@@ -0,0 +1,211 @@
...
+  font-family: "GenB";
+  src: url(../fonts/GenB.ttf);
+}
+
+body, html {
+  margin: 0;
+  padding: 0;
+}
+
+
```
Comment (magicsd) [inline resources/css/index.css]: Unnecessary blank line

**Comment by katsu223** [comment]
katsu223.surge.sh/

File: index.html [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+    <div class="container">
+      <div class="shadow"></div>
+      <div class="header">
+        <div class="logo">
+          <h1 class="logo-title"><a class="logo-link" href="#main-page">Lambda</a></h1>
+        </div>
+        <nav>
+          <div class="nav">
+            <ul class="main-menu">
+              <li class="menu-button"><a class="menu-link" href="#home">Home</a></li>
```
Comment (magicsd) [inline index.html]: It's easier to read
```suggestion
              <li class="menu-button">
                <a class="menu-link" href="#home">Home</a>
              </li>
```

File: index.html [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+              <li class="menu-button"><a class="menu-link" href="#about">About</a></li>
+              <li class="menu-button"><a class="menu-link" href="#igredients">Igredients</a></li>
+              <li class="menu-button"><a class="menu-link" href="#menu">Menu</a></li>
+              <li class="menu-button"><a class="menu-link" href="#reviews">Reviews</a></li>
+              <li class="menu-button"><a class="menu-link" href="#reservations">Reservations</a></li>
+            </ul>
+          </div>
+          <div class="socials">
+            <ul class="social-menu">
+              <li class="social-button"><a class="social-link" href="#twitter"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
```
Comment (magicsd) [inline index.html]:
```suggestion
              <li class="social-button">
                <a class="social-link" href="#twitter">
                  <i class="fab fa-twitter" aria-hidden="true"></i>
                </a>
              </li>
```

File: resources/css/index.css [hunk 28]
```diff
@@ -19,193 +18,167 @@ body, html {
...
-  color: #fff;
+.divider {
+  display: inline-block;
+  margin-top: 30px;
 }
 
-.layout-positioner {
-  z-index: 3;
+.logo {
+  float: left;
```
Comment (magicsd) [inline resources/css/index.css]: It's not a good practice to use floats

File: resources/css/index.css [hunk 29]
```diff
@@ -19,193 +18,167 @@ body, html {
...
+  margin: 0;
 }
 
-.layout-positioner .header {
-  margin: 35px 0;
+.logo-link {
+  color: #fff;
+  font-family: "Yeseva One", sans-serif;
+  font-size: 36px;
+  font-weight: 400;
```
Comment (magicsd) [inline resources/css/index.css]: It's a default weight, should be removed

File: resources/css/index.css [hunk 30]
```diff
@@ -19,193 +18,167 @@ body, html {
...
-  text-align: center;
-  font-size: 90px;
-  color: #fff;
-  margin: 0;
-  font-weight: 400;
-  text-shadow:  1px 1px 2px #000;
+.button-menu {
+  border: 1px solid #fff;
+  background-color: rgba(255, 255, 255, 0.2);
+  margin-left: -5px;
```
Comment (magicsd) [inline resources/css/index.css]: Something went wrong here

File: resources/css/index.css [hunk 31]
```diff
@@ -19,193 +18,167 @@ body, html {
...
 }
 
 .container {
-  position: relative;
-  z-index: 2;
+  background-attachment: fixed;
```
Comment (magicsd) [inline resources/css/index.css]: I don't think it's necessary, imho

File: resources/css/index.css [hunk 32]
```diff
@@ -19,193 +18,167 @@ body, html {
...
-.layout-positioner::after {
-  content: "";
-  display: table;
-  clear: both;
- }
-
- .navbar {
-  float: right;
+.nav {
+  float: left;
```
Comment (magicsd) [inline resources/css/index.css]: It's not a good practice to use floats

File: resources/css/index.css [hunk 33]
```diff
@@ -19,193 +18,167 @@ body, html {
...
   padding: 0;
  }
 
- .navbar .main-menu li {
+.main-menu .menu-button {
+  box-sizing: border-box;
   display: inline-block;
   margin:0px;
-  box-sizing: border-box;
   margin-left: -4px;
```
Comment (magicsd) [inline resources/css/index.css]: Something went wrong here as well

File: resources/css/index.css [hunk 34]
```diff
@@ -19,193 +18,167 @@ body, html {
...
   border-top: 5px solid #c9c6ba;
   padding-top: 39px;
  }
 
- .social-media {
-  float: right;
+ .header::after {
+  clear: both;
+  content: "";
+  display: table;
```
Comment (magicsd) [inline resources/css/index.css]: Something went wrong here

File: resources/css/index.css [hunk 35]
```diff
@@ -19,193 +18,167 @@ body, html {
...
+ }
+
+.header {
+  position: relative;
+  z-index: 2;
+}
+
+ .socials {
+  float: left;
```
Comment (magicsd) [inline resources/css/index.css]: Hmmm

File: resources/css/index.css [hunk 36]
```diff
@@ -19,193 +18,167 @@ body, html {
...
+  box-shadow: 0px 50px 119px 0px rgba(0,0,0,0.67);
+  height: 160px;
+  position: absolute;
+  top: -160px;
+  width: 1600px;
+}
+
+.intro {
+  text-align: center;
+  margin-top: 110px;
```
Comment (magicsd) [inline resources/css/index.css]: Seems like parent component's padding should do this job

File: resources/css/index.css [hunk 37]
```diff
@@ -19,193 +18,167 @@ body, html {
...
   color: #f0c955;
   text-shadow: 1px 1px 2px #000;
  }
 
- .social-media .social-menu li:first-child {
+ .social-button:first-child {
   margin-left: 0px;
  }
+
+ .shadow {
```
Comment (magicsd) [inline resources/css/index.css]: Try to position it better

File: index.html (hunk 26 above)
Comment (katsu223) [reply index.html]: fixed

File: index.html (hunk 27 above)
Comment (katsu223) [reply index.html]: fixed

File: resources/css/index.css (hunk 28 above)
Comment (katsu223) [reply resources/css/index.css]: fixed

File: resources/css/index.css (hunk 29 above)
Comment (katsu223) [reply resources/css/index.css]: Without this property the logo font is too bold.

File: resources/css/index.css (hunk 30 above)
Comment (katsu223) [reply resources/css/index.css]: What's wrong? I deleted the whitespace between inline-blocks.

File: resources/css/index.css (hunk 31 above)
Comment (katsu223) [reply resources/css/index.css]: I think it adds some dynamic effects at the page.

File: resources/css/index.css (hunk 32 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 33 above)
Comment (katsu223) [reply resources/css/index.css]: Deleted whitespace.

File: resources/css/index.css (hunk 34 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 35 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 36 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 37 above)
Comment (katsu223) [reply resources/css/index.css]: How should I position it?

**Comment by katsu223** [comment]
npm i -g live-server && live-server

**Comment by katsu223** [comment]
katsu223.surge.sh

File: resources/css/index.css [hunk 38]
```diff
@@ -0,0 +1,186 @@
...
+  box-sizing: border-box;
+  color: #fff;
+  display: inline-block;
+  font-size: 18px;
+  font-family: "Lato", sans-serif;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px 35px;
+  margin-top: 30px;
```
Comment (magicsd) [inline resources/css/index.css]: I'd put buttons into `<div class="button-group"></div>` and add margin to this container. Or add `margin-bottom` to `.divider`.

File: resources/css/index.css [hunk 39]
```diff
@@ -0,0 +1,186 @@
...
+  border: 1px solid #fff;
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.container {
+  background-attachment: fixed;
+  background-image: url(../images/background/Bg.png);
+  background-position: 50% 0px;
+  height: 750px;
+  width: 1600px;
```
Comment (magicsd) [inline resources/css/index.css]: You've already defined width
```suggestion
  width: 100%;
```

File: resources/css/index.css [hunk 40]
```diff
@@ -0,0 +1,186 @@
...
+.button-menu {
+  border: 1px solid #fff;
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.container {
+  background-attachment: fixed;
+  background-image: url(../images/background/Bg.png);
+  background-position: 50% 0px;
+  height: 750px;
```
Comment (magicsd) [inline resources/css/index.css]: Is it necessary?

File: resources/css/index.css [hunk 41]
```diff
@@ -0,0 +1,186 @@
...
+
+.logo {
+  display: inline-block;
+  vertical-align: top;
+  padding-top: 35px;
+  padding-left: 175px;
+}
+
+.logo-title {
+  margin: 0;
```
Comment (magicsd) [inline resources/css/index.css]: Will be unnecessary after dropping margins above.

File: resources/css/index.css [hunk 42]
```diff
@@ -0,0 +1,186 @@
...
+.nav-bar {
+  display: inline-block;
+ }
+
+ .nav {
+  padding-left: 210px;
+ }
+
+.main-menu {
+  font-size: 0;
```
Comment (magicsd) [inline resources/css/index.css]: Is it useful?

File: resources/css/index.css [hunk 43]
```diff
@@ -0,0 +1,186 @@
...
+  list-style: none;
+  margin: 0;
+  padding: 0;
```
Comment (magicsd) [inline resources/css/index.css]: These two lines above will be unnecessary

File: resources/css/index.css [hunk 44]
```diff
@@ -0,0 +1,186 @@
...
+ }
+
+.main-menu .menu-button {
+  box-sizing: border-box;
+  display: inline-block;
+  margin:0px;
```
Comment (magicsd) [inline resources/css/index.css]: You already know 😄

File: resources/css/index.css [hunk 45]
```diff
@@ -0,0 +1,186 @@
...
+
+.main-menu {
+  font-size: 0;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+ }
+
+.main-menu .menu-button {
+  box-sizing: border-box;
```
Comment (magicsd) [inline resources/css/index.css]: Should it be here?

File: resources/css/index.css [hunk 46]
```diff
@@ -0,0 +1,186 @@
...
+  display: inline-block;
+  margin:0px;
+ }
+
+ .main-menu .menu-button:first-child {
```
Comment (magicsd) [inline resources/css/index.css]: What is the purpose?

File: resources/css/index.css [hunk 47]
```diff
@@ -0,0 +1,186 @@
...
+  margin-left: 0;
+ }
+
+.menu-button .menu-link {
+  color: #fff;
+  display: block;
+  font-family: "GenB", sans-serif;
+  font-size: 21px;
+  overflow: hidden;
```
Comment (magicsd) [inline resources/css/index.css]: What is the purpose?

File: resources/css/index.css [hunk 48]
```diff
@@ -0,0 +1,186 @@
...
+.socials .social-menu {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+ }
+
+.social-button {
+  box-sizing: border-box;
+  display: inline-block;
+  margin:0px;
```
Comment (magicsd) [inline resources/css/index.css]: ?

File: resources/css/index.css [hunk 49]
```diff
@@ -0,0 +1,186 @@
...
+  margin-left: -4px;
```
Comment (magicsd) [inline resources/css/index.css]: ?

File: resources/css/index.css [hunk 50]
```diff
@@ -0,0 +1,186 @@
...
+ }
+
+.socials .social-menu {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+ }
+
+.social-button {
+  box-sizing: border-box;
```
Comment (magicsd) [inline resources/css/index.css]: ?

File: resources/css/index.css [hunk 51]
```diff
@@ -0,0 +1,186 @@
...
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav,
+.nav-bar {
+  display: inline-block;
+ }
+
+ .nav {
```
[the remark this answers is not in the document]
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css [hunk 52]
```diff
@@ -0,0 +1,186 @@
...
+  padding-left: 210px;
+ }
+
+.main-menu {
+  font-size: 0;
+  list-style: none;
```
[the remark this answers is not in the document]
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css [hunk 53]
```diff
@@ -0,0 +1,186 @@
...
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+
+@font-face {
+  font-family: "GenB";
+  src: url(../fonts/GenB.ttf);
+}
+
+body, html {
```
[the remark this answers is not in the document]
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css [hunk 54]
```diff
@@ -0,0 +1,186 @@
...
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.button {
+  box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css [hunk 55]
```diff
@@ -0,0 +1,186 @@
...
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px 35px;
+  margin-top: 30px;
+}
+
+.button-book {
+  border: 1px solid black;
+  background-color: rgba(0, 0, 0, 0.6);
```
[the remark this answers is not in the document]
Comment (katsu223) [reply resources/css/index.css]: Fixed here and below.

File: resources/css/index.css (hunk 38 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 39 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 40 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 41 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed

File: resources/css/index.css (hunk 42 above)
Comment (katsu223) [reply resources/css/index.css]: Deleted whitespace between buttons in navigation menu according to the task

File: resources/css/index.css (hunk 43 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 44 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 45 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 46 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 47 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 48 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 49 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 50 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed

File: resources/css/index.css (hunk 42 above)
Comment (magicsd) [reply resources/css/index.css]: In this case it's not obligatory, but I don't mind

**Pull request merged**